rust/src/rfb/parser.rs

```diff
@@ -24,7 +24,7 @@ use nom7::*;
 use std::fmt;
 use std::str;
 
-#[derive(Debug,PartialEq)]
+#[derive(Debug,PartialEq, Eq)]
```

Comment:
Could the commit message explain why we need this ?
Or squash it in another commit

Reply:
Will squash. This was needed for a different place in the code. Can't remember why but I get the point -- if I can't then it might be pointless to have separately.